### DISCUSSION TITLE
Show Frame number

### DISCUSSION THREAD

**Discussion by orena1** [bonsai-rx/bonsai#915] [Ideas]
Hi, is it possible to show the frame number of the screen?
It might be helpful for debugging.

Maybe also other info as well, something like youtube "Stats for nerds"
![image](https://user-images.githubusercontent.com/8983713/171267117-e21fbbc9-f798-47a7-928c-ba8c0d8b8b9c.png)

Thanks

**Comment by bruno-f-cruz** [comment]
You can probably infer this if you take the difference between `UpdateFrame` and `RenderFrame` events.
[Stackoverflow](https://stackoverflow.com/questions/23542591/open-tk-difference-onrenderframe-and-onupdateframe#:~:text=renderFrame%20is%20the%20main%20loop,or%2060%20times%2Fsecond).

**Reply by orena1** [reply]
Thanks, but there is no way to do it now? I moved it to ideas

**Reply by bruno-f-cruz** [reply]
I meant something along these lines. You count the number of events from `RenderFrame` and from `UpdateFrame`. If your target (defined in `CreateWindow`) is the same for both (e.g. 60Hz), mismatches in such a counter will likely correspond to dropped frames.
![image](https://user-images.githubusercontent.com/7049351/172316923-640deaae-3cc7-42d1-8d76-b17e8a8919e6.png)
